<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>math5a_bezierForm</class>
 <widget class="QMainWindow" name="math5a_bezierForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fractales 3D - ESGI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="centralLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Configuration</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupMode">
       <property name="title">
        <string>Mode de génération des points</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="6" column="1" colspan="2">
         <widget class="QPushButton" name="bResetData">
          <property name="text">
           <string>Réinitialiser les données</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QLabel" name="laTimeCalcSurface">
          <property name="text">
           <string>0 us</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbShowTexture">
       <property name="text">
        <string>Texture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbShowPoints">
       <property name="text">
        <string>Afficher les points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbShowGrid">
       <property name="text">
        <string>Grille et les axes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupLight">
       <property name="title">
        <string>Lumières</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="6" column="0">
         <widget class="QCheckBox" name="cbDiffuse">
          <property name="text">
           <string>Diffuse</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="bColorS1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="cbSpecular">
          <property name="text">
           <string>Specular</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="cbLight1">
          <property name="text">
           <string>Source 1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="bColorS2">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="cbLight2">
          <property name="text">
           <string>Source 2</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="bColorObj">
          <property name="text">
           <string>Couleur du patch</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>bColorS1</zorder>
       <zorder>bColorS2</zorder>
       <zorder>cbLight1</zorder>
       <zorder>cbLight2</zorder>
       <zorder>cbDiffuse</zorder>
       <zorder>cbSpecular</zorder>
       <zorder>bColorObj</zorder>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupRotation">
       <property name="title">
        <string>Rotation</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="spinZ">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>359.899999999999977</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="spinY">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>359.899999999999977</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spinX">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>359.899999999999977</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="rbRotObj">
          <property name="text">
           <string>Objet</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="rbRotCam">
          <property name="text">
           <string>Caméra</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bResetCam">
       <property name="text">
        <string>Réinitialiser le caméra</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bQuit">
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
